### PR TITLE
Update fileapi example to work with Express 4

### PR DESCRIPTION
**Pull request by hodannyy**
Express does not inherit from http in newer versions so express.createServer() is deprecated.

### CODE REVIEW

**Comment by lpinca** [comment]
Thanks. How about tweaking the example to make it work with express 4 instead?
It shouldn't be much work.

**Comment by hodannyy** [comment]
Sounds good. I've updated the fileapi example to work with Express 4.

File: examples/fileapi-express_4/server.js [hunk 1]
```diff
@@ -101,6 +102,6 @@ fs.mkdir(path.join(__dirname, '/uploaded'), function () {
   // ignore errors, most likely means directory exists
   console.log('Uploaded files will be saved to %s/uploaded.', __dirname);
   console.log('Remember to wipe this directory if you upload lots and lots.');
-  app.listen(8080);
+  server.listen(8080);
```
Comment (lpinca) [inline examples/fileapi-express_4/server.js]: Can you add a callback for the `listening` event and move the next line (`console.log()`) there?

**Comment by lpinca** [comment]
@hodannyy I think there is no need to rename the folder, please keep the original name.

**Comment by hodannyy** [comment]
@lpinca I've renamed it back to the original name. Thought it was a naming convention since the Express 3 serverstats example has an express_3 prefix.

**Comment by lpinca** [comment]
I see, I think it's better to merge those two and keep only one `serverstats` example which uses the latest express.

**Comment by lpinca** [comment]
Thanks!

**Pull request merged**